Resources/config/user_context.xml

```diff
@@ -39,5 +40,9 @@
             <argument />
             <argument />
         </service>
+
+        <service id="fos_http_cache.user_context.anonymous_request_matcher" class="FOS\HttpCacheBundle\UserContext\AnonymousRequestMatcher">
+            <argument />
```

Comment:
> i think tihs is already asked for in the configuration

did you have something specific in mind ? I'm not sure what variable should actually be used there.